contracts/XERC20/XERC20.sol

```diff
@@ -0,0 +1,286 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.4 <0.9.0;
+
+import { IXERC20 } from "./interfaces/IXERC20.sol";
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import { ERC20Permit } from "@openzeppelin/contracts/token/ERC20/extensions/draft-ERC20Permit.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @custom:attribution https://github.com/defi-wonderland/xERC20/blob/dev/solidity/contracts/XERC20.sol
+ */
+contract XERC20 is ERC20, Ownable, IXERC20, ERC20Permit {
```

Comment:
Do we want to keep our own versions (is there even a reason) or is it better just to import from mainline?

Reply:
This isn't published anywhere (like npm) and the code is audited I woudl keep it as is.

Reply:
If it is like that I would rather consider a fork a add npm stuff and then we can easily pull changes from origin repository. 
This way we would always have to copy paste stuff for any changes.

Reply:
This is usually the way to go and we did it like that for all the 3rd party contracts we use.
I don't see any changes happening very often, in the last almost year there was only one and it was only a naming improvement.